src/vulkan/pipeline.cc

```diff
@@ -65,87 +65,107 @@ void Pipeline::Shutdown() {
   if (r.IsSuccess())
     command_->SubmitAndReset(fence_timeout_ms_);
 
-  DestoryDescriptorPools();
-  DestoryDescriptorSetLayouts();
+  for (size_t desc = 0; desc < descriptor_sets_.size(); ++desc) {
```

Comment:
Let's change desc to i. Naming it desc makes me think it's the descriptor, not the index.

Reply:
done